src/components/Promptbar/components/Prompt.tsx

```diff
@@ -191,7 +191,12 @@ export const PromptComponent = ({ item: prompt, level }: Props) => {
       >
         <IconBulb size={18} className="text-gray-500" />
 
-        <div className="relative max-h-5 flex-1 truncate break-all pr-4 text-left">
+        <div
+          className={classNames(
+            isDeleting ? 'mr-12' : 'pr-4',
```

Comment:
Why are you using different approach together? Is it possible to use only margin or only padding?

Reply:
fixed